development/Adding-a-new-platform-source.md

```diff
@@ -1,17 +1,17 @@
-Thank you for help for the development of *Chat Overflow*.  
-While plugins live in their own project, source connections (e.g. to platform like Twitch or Discord) exist in the framework itself.  
-By this, we can make sure that everyone profits from the new possibilites.  
+Thank you for helping developing *Chat Overflow*.  
```

Comment:
Still sounds a bit weird, what about "thank you for your help in chat overflow development"

Reply:
What would you think of "Thank you for improving *Chat Overflow*"?